acc.php

```diff
@@ -1355,7 +1355,7 @@
 		}
 
 		Notification::requestClosed($request, $crea);
-		BootstrapSkin::displayAlertBox("Request " . $request->getId() . " (" . htmlentities($request->getName(), ENT_COMPAT, 'UTF-8') . ") marked as 'Done'.", "alert-success");
+		BootstrapSkin::displayAlertBox("Request " . $request->getId() . " (" . htmlentities($request->getName(), ENT_COMPAT, 'UTF-8') . ") marked as '" . $crea . "'.", "alert-success");
```

Comment:
There's a possibility for stored XSS here. Please can you wrap the use of `$crea` in a call to `htmlentities`?

```suggestion
		BootstrapSkin::displayAlertBox("Request " . $request->getId() . " (" . htmlentities($request->getName(), ENT_COMPAT, 'UTF-8') . ") marked as '" . htmlentities($crea, ENT_COMPAT, 'UTF-8') . "'.", "alert-success");
```